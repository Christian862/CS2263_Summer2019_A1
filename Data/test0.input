u 4
u 6
u 7
o
o
o
x