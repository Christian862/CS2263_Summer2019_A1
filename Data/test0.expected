4
6
7
7
6
4
Successfully executed 6 instructions
0 elements
